packaging/scripts/run-cups-browsed

```diff
@@ -0,0 +1,110 @@
+#! /bin/sh
```

Comment:
There needs to be a way to disable cups-browsed.

Reply:
The best way is to move cups-browsed into its own Snap.

Generally it will go away with CUPS 3.x, but for the short remaining life time of CUPS 2.x we should do something about it.

cups-browsed in its own Snap would be the most elegant solution, do not install that Snap if all your print dialogs are nicely doing CPDB and so cope with temporary printers, install it if you still have to cope with broken print dialogs.

If we keep cups-browsed in the CUPS Snap, we need a way to make it easier than `sudo snap disable cups.cups-browsed` for the users of the CUPS Snap to turn off cups-browsed.